test/Microsoft.Extensions.CommandLineUtils.Tests/ArgumentEscaperTests.cs

```diff
@@ -17,6 +17,8 @@ public class ArgumentEscaperTests
         [InlineData(new[] { @"C:\Program Files\dotnet\" }, @"""C:\Program Files\dotnet\\""")]
         [InlineData(new[] { @"backslash\""preceedingquote" }, @"backslash\\\""preceedingquote")]
         public void EscapesArguments(string[] args, string expected)
-            => Assert.Equal(expected, ArgumentEscaper.EscapeAndConcatenate(args));
+        {
```

Comment:
This is what was triggering the analyzer exception. They already have a PR to fix it: https://github.com/xunit/xunit.analyzers/pull/77, but it's easy for us to just workaround it by using braces instead.